### ISSUE TITLE
Support the `SPDX-FileCopyrightText` keyword explicitly in the copyright agent

### ISSUE THREAD

**Issue by maxhbr** [fossology/fossology#1513]
The copyright agent should especially look for the `SPDX-FileCopyrightText` Keyword and recognize it as copyright. See also: https://reuse.software/

**Comment by silverhook** [comment]
I agree, a REUSE agent would be super useful and should be rather easy to implement (compared to others), as it just needs to look for two very simple regexps.

**Comment by NicolasToussaint** [comment]
Since we're working on the integration of other parts of REUSE.Software specifications, we can also take care of the SPDX Copyright keyword.

**Comment by shaheemazmalmmd** [comment]
Recommended changes are already added please see https://github.com/fossology/fossology/blob/master/src/copyright/agent/copyright.conf#L29
